src/linear_algebra/basic.lean

```diff
@@ -188,6 +188,10 @@ lemma sum_apply (t : finset ι) (f : ι → M →ₗ[R] M₂) (b : M) :
   (∑ d in t, f d) b = ∑ d in t, f d b :=
 (t.sum_hom (λ g : M →ₗ[R] M₂, g b)).symm
 
+@[simp] lemma coe_sum (t : finset ι) (f : ι → M →ₗ[R] M₂) :
```

Comment:
I thought I already added this lemma (perhaps in another file)... I guess library search couldn't find it?

Reply:
`library_search` times out, and mathlib docs don't know about this lemma (may be you used some other name?).

Reply:
Yes, this lemma already exists but is called `coe_fn_sum`. Perhaps delete the other one?

Reply:
Thanks! Added `@[simp]` to `coe_fn_sum`